### ISSUE TITLE
udp broadcast documentation should link to the interface instead of to a concrete implementation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6941]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#6941|
|Type|enhancement|
|Created|2014-02-02 13:11:08Z|
|Branch|https://github.com/twisted/twisted/tree/udp-docs-6941|

`udp.rst` says:

   Broadcast is enabled in Twisted's UDP implementation by passing ``True`` to :api:`twisted.internet.udp.Port.setBroadcastAllowed <setBroadcastAllowed>`

Instead, the link should be to `twisted.internet.interfaces.IUDPTransport.setBroadcastAllowed`.

`twisted.internet.udp` is an implementation detail of the POSIX-based reactors.

<details><summary>Searchable metadata</summary>

```
trac-id__6941 6941
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_udp_docs_6941 branches/udp-docs-6941
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1391346668000000 1391346668000000
changetime__1391359529000000 1391359529000000
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [41517]) Branching to udp-docs-6941.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl set owner to @hawkowl|
|-|-|

I'll take this.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl removed owner|
|-|-|

Fixed this up, putting it up for review.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_4"></a>|@hynek set owner to @hawkowl|
|-|-|

The bots are green and I’ve hand-verified the api paths. Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [41525]) Merge udp-docs-6941: UDP Broadcast documentation should link to the interface instead of to a concrete implementation

Author: hawkowl
Reviewer: hynek
Fixes: [#6941](https://github.com/twisted/twisted/issues/6941)
